### DISCUSSION TITLE
Release notes for 2.7.4

### DISCUSSION THREAD

**Discussion by LorenzoBettini** [eclipse-tycho/tycho#1113] [Q&A] [answered]
Hi

Tycho 2.7.4 was released a few days ago, but I don't see anything about this release in the release notes.

**Comment by laeubi** [comment, marked as answer]
The releases release notes are here: https://github.com/eclipse/tycho/blob/tycho-2.7.4/RELEASE_NOTES.md#274 and will be integrated soon in the master branch as well: https://github.com/eclipse/tycho/pull/1109
